src/service/extension-location.js

```diff
@@ -37,21 +37,23 @@ function calculateScriptBaseUrl(location, isLocalDev, isTest) {
  * Calculate script url for an extension.
  * @param {!Location} location The window's location
  * @param {string} extensionId
+ * @param {string} extensionVer
```

Comment:
Done. Just FYI: I also do this upstream, but I suppose it can't hurt to do this here also.